Lib/glyphsLib/classes.py

```diff
@@ -1234,8 +1239,20 @@ def shouldWriteValueForKey(self, key):
         if key in ("xHeight", "capHeight", "ascender"):
             # Always write those values
             return True
+        if key == "name":
+            if self.font and int(self.font.appVersion) >= 1075:
```

Comment:
if the appVersion type is a string, instead of simply an integer, I guess the reason is that it may (or may have) contained some non-numeric characters in it. Unconditionally casting it to int may fail with ValueError if the string contains non-numeric characters.  
Question for @schriftgestalt: does "appVersion" string always only contain numbers? If so, why do you have to type it as a string? On the other hand, if it _may_ also contain non-numbers, what exactly is the format of "appVersion", so that we can have a function that converts it into a tuple in order to correctly do comparison between them?